CollAction/Frontend/app/project/ProjectFilter.tsx

```diff
@@ -25,7 +25,7 @@ export class ProjectFilter extends React.Component<IProjectFilterProps, IProject
     super(props);
 
     const statusList: IDropDownListItem[] = [
-      { id: "1", name: "Open" },
+      { id: "1", name: "Open & Closed" },
```

Comment:
The find project layout is a bit fragile. Do make sure this doesn't introduce line wrapping when testing.